app/routes/404.tsx

```diff
@@ -1,15 +1,20 @@
 import * as React from 'react'
-import {ErrorPage} from '../components/errors'
+import {HeroSection} from '../components/sections/hero-section'
+import {images} from '../images'
 
 export function meta() {
   return {title: "Ain't nothing here"}
 }
 
 export default function NotFoundPage() {
   return (
-    <ErrorPage
-      title="404 - Oh no, how did you get here?"
-      subtitle="This is not a page on kentcdodds.com. So sorry."
-    />
+    <main>
```

Comment:
This was a bit unexpected, but when I use the `useRequestInfo` in the 404 (via error > article recommendations), an error was thrown mentioning that the component needs to be wrapped in `RequestInfoProvider`. After adding that, I ended up with an error mentioning that I need to run in a `remix route element`.

I didn't think it's worth going down this rabbit hole, so I replaced the root 404 with something simpler.

Reply:
Yeah, I actually ran into a similar problem when I tried to remove the article fixtures. I'll look into it later. Thanks!